content/_code-samples/require-destination-tags/py/require-destination-tags.py

```diff
@@ -0,0 +1,49 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models.transactions import AccountSet, AccountSetFlag
+from xrpl.transaction import safe_sign_and_autofill_transaction, submit_transaction
+from xrpl.wallet import generate_faucet_wallet
+from xrpl.models.requests import AccountInfo
+
+# Stand-alone code sample for the "Require Destination Tags" tutorial:
+# https://xrpl.org/require-destination-tags.html
+
+if __name__ == "__main__":
```

Comment:
I personally would not define that for an example code since this line brings no advantage to the asked use case. This line is addressing an issue of python. If you want to read more about it, this explains it pretty well: https://stackoverflow.com/a/419185